pages/osx/port.md

```diff
@@ -0,0 +1,27 @@
+# port
+
+> Package manager for macOS.
+
+- Search for a package:
+
+`port search {{search_term}}`
+
+- Install a package:
+
+`sudo port install {{package_name}}`
+
+- List installed packages:
+
+`port installed`
+
+- Update port and fetch latest list of available packages:
+
+`sudo port selfupdate`
+
+- Upgrade outdated packages:
+
+`sudo port upgrade outdated`
+
+- Remove inactive package versions:
```

Comment:
What does `inactive` mean in this context?

Reply:
port does not remove older versions of installed packages after an upgrade, so in case an update goes wrong you can revert to the already installed versions. These older versions are referred to as inactive by the command. 

Perhaps obsolete or superseded might be clearer? Or something like:

`- Remove old versions of installed packages `

Reply:
Ah, I see. Yeah, that's a much better way of putting it :smiley_cat: